tests/end_to_end/server_test.py

```diff
@@ -48,11 +46,7 @@ def run(self, *args, **kwargs):
             self.otherTeardown()
 
     def runClientCmd(self, client, command):
-        try:
-            client.runCommand(command)
-        except subprocess.CalledProcessError as error:
-            self.server.printDebugInfo()
-            raise error
+        client.runCommand(command)
```

Comment:
But forces debug output on us, whether we want it or not. Why not leave that decision up to the user of the api?